components/ReviewSection.tsx

```diff
@@ -10,26 +10,28 @@ import { SectionRow } from "../helpers/useReviewSectionRows";
 
 interface Props {
   heading: string;
+  loading: boolean;
   rows?: SectionRow[];
 }
 
 export const ReviewSection: React.FunctionComponent<Props> = (props) => {
-  const { heading, rows } = props || {};
+  const { heading, loading, rows } = props || {};
 
   return (
     <>
       <Heading level={HeadingLevels.H2}>{heading}</Heading>
-      {rows && rows.length ? (
-        <SummaryList rows={rows} />
-      ) : (
+      {loading ? (
         <Paragraph>Loading...</Paragraph>
+      ) : (
+        rows && rows.length > 0 && <SummaryList rows={rows} />
```

Comment:
Is `rows.length` not falsy when it's 0? I feel I've put this a few times so would be good to know!

Reply:
It is, but it's also `0`, so React would render it.